prometheus/scripts/start-alertmanager.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Script to start alert manager.
+
+# Alertmanager doesn't provide a way to replace config variables, and we need
+# to write our own script to do this. See
+# https://github.com/prometheus/alertmanager/issues/504
+
+sed "s|\$ALERTMANAGER_SLACK_API_URL|$ALERTMANAGER_SLACK_API_URL|g" /config/alertmanager.yml > /tmp/alertmanager.yml  # Permissions to write to the same file is a problem, so we write a /tmp file
```

Comment:
Is the `/config/alertmanager.yml` path in the default docker image? I don't see it mounted anywhere, hence my confusion.

Reply:
It's mounted in the `docker-compose.yml` file.

Reply:
Ah yes! i see it :)) thanks!